src/Microsoft.SqlTools.ServiceLayer/Microsoft.SqlTools.ServiceLayer.csproj

```diff
@@ -70,13 +71,11 @@
 	<ItemGroup>
 		<EmbeddedResource Include=".\Agent\NotebookResources\NotebookJobScript.ps1" />
 	</ItemGroup>
-	<!-- this target enables dependency files to be copied as part of the output of ProjectReference.
-	https://github.com/dotnet/sdk/issues/1675
-	 -->
-	<Target Name="AddRuntimeDependenciesToContent" Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'" BeforeTargets="GetCopyToOutputDirectoryItems" DependsOnTargets="GenerateBuildDependencyFile;&#xD;&#xA;							GenerateBuildRuntimeConfigurationFiles">
```

Comment:
This doesn't seem to be necessary when adding the SourceLink package - this was causing an error on publish since the build itself was also copying the deps.json file. So removing this since the SourceLink package is handling doing the copying for us.